ble/UUID.h

```diff
@@ -42,7 +97,7 @@ class UUID {
      *         different service or characteristics on the BLE device.
      *
      * @param longUUID
-     *          The 128-bit (16-byte) UUID value, MSB first (big-endian).
+     *          The 128-bit (16-byte) UUID value, LSB first (little-endian).
```

Comment:
@marcuschangarm Perhaps the answer to our situation is that we leave the existing constructor unchanged, but offer an alternate constructor which accepts UUIDs in little-endian form. We should also make clear that the internal representation is little-endian; and this will be exposed to the caller when using `getBaseUUID`.